src/modules/auth/components/AppLogo.tsx

```diff
@@ -0,0 +1,22 @@
+import Apple from 'src/assets/apple.svg';
+import Andriod from 'src/assets/andriod.svg';
+import Globe from 'src/assets/globe.svg';
+
+export function AppLogo(): JSX.Element {
```

Comment:
`AppLogo` is not the best name for this... Since it sounds more like our app logo.
My recommendation for this component would be: `SupportedPlatformsList`...

Reply:
Also, let's follow the convention for naming component files. 
Eg. for this Component: `AppLogo.component.tsx` 
"Rule" for naming files: `<name>.<singularFolderName>.<fileType>` 
Having a convention for naming files will help us navigate the project and use IDE search more efficiently.
Especially if you have files with the "same" name but for different stuff. 
Eg. We can have `Auth.page.tsx` and also `Auth.router.tsx`...